etc/requirements/base.txt

```diff
@@ -24,14 +24,14 @@ redis==3.5.3
 gunicorn==20.1.0
 
 # Docker
-container_inspector>=3.1.2
+container_inspector>=21.5.25
```

Comment:
Using `>=` with CalVer is problematic for API changes, see https://github.com/nexB/scancode.io/pull/181/commits/7b7fc2b11dd15a7cdae8d84912ce76e02d53f761
We should revert back to exact versions.

Reply:
That's a container-inspector issue ... and that's my making: this ship has sailed there now and I cannot see how I can revert there.
Note that the API for this container-inspector library is fast evolving but I will thrive to keep it backward compatible as much as possible.